_posts/commands/2020-05-01-mom_practice.md

```diff
@@ -6,7 +6,11 @@ tags:
 cvar_ref: mom_practice_safeguard
 ---
 
-Toggles practice mode, allowing the player to fly around in noclip during a run. Teleports the player back upon untoggling.
+Toggles practice mode, allowing the player to fly around in noclip. 
+The timer will continue to tick if toggled during in a run.
+When toggled off, the player is teleported back to their position.
```

Comment:
When toggled off while not in a run, they are not teleported anywhere and keep their movement from the last tick the player had in practice mode.